language/move-ir/types/src/ast.rs

```diff
@@ -133,6 +135,17 @@ pub struct ImportDefinition {
     pub alias: ModuleName,
 }
 
+//**************************************************************************************************
+// Friends
+//**************************************************************************************************
+
+/// A friend declaration
+#[derive(Clone, Debug, PartialEq)]
+pub struct FriendDeclaration {
+    /// the friend
+    pub ident: ModuleIdent,
+}
+
```

Comment:
I'd remove this. There is no point in having this over just having a module ident

Reply:
Removed.